src/AggregatorStack.ts

```diff
@@ -80,15 +92,27 @@ class Notifier extends Construct {
 
   }
 
-  setupMonitoringFunction(prefix: string) {
-    const lambda = new MonitoringFunction(this, 'slack-lambda');
+  setupMonitoringFunction(prefix: string, branchName: string) {
+    const lambda = new MonitoringFunction(this, 'slack-lambda', {
+      environment: {
+        BRANCH_NAME: branchName,
+      },
+      description: `Slack notification lambda (${prefix})`,
+    });
     for (const priority of Statics.monitoringPriorities) {
       const paramValue = StringParameter.valueForStringParameter(this, `${Statics.ssmSlackWebhookUrlPriorityPrefix}-${prefix}-${priority}`);
       lambda.addEnvironment(`SLACK_WEBHOOK_URL_${priority.toUpperCase()}`, paramValue);
     }
     this.subscribeLambda(lambda);
   }
 
+  setupLogQueryJob(prefix: string, brancName: string) {
+    new LogQueryJob(this, 'log-query-job', {
+      prefix: prefix,
+      branchName: brancName,
+      deployToEnvironments: getConfiguration(brancName).deployToEnvironments,
```

Comment:
brancName nog even naar branchName wijzigen?

Reply:
Dit In #130 ook gefixt

Reply:
Ziet er goed uit, merged!